src/App.css

```diff
@@ -0,0 +1,7 @@
+.CoffeeShopsView__container {
+  margin: 20px 6vw 0 6vw;
+}
+
+.CoffeeShopsView__error {
+  color: red;
+}
```

Comment:
I'm curious what naming convention is this? Do you follow any?

Reply:
I follow the BEM naming convention.

I see now that CoffeeShopsView__error could have been just 'error', as it's a reusable class.

Reply:
You can keep it for now as a part of the `CoffeeShopsView`, until you actually need it to be reused. My question arose from the letter casing you were using, I was familiar with BEM but is `PascalCase` normal with BEM?

Reply:
Actually, I'll change it to coffee-shops-view__container.